text/0006-token-api.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: token_api
+- Start Date: 2021-08-04
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
+
+# Summary
+[summary]: #summary
+
+Token API will provide token information by requesting `hathor-wallet-service`.
```

Comment:
I guess we don't have to explain the low level in the summary. Maybe explain where it will be used.